extensions/amp-consent/0.1/consent-info.js

```diff
@@ -355,6 +362,10 @@ export function composeMetadataStoreValue(consentInfoMetadata) {
     storageMetadata[METADATA_STORAGE_KEY.CONSENT_STRING_TYPE] =
       consentInfoMetadata['consentStringType'];
   }
+  if (consentInfoMetadata['additionalConsent']) {
```

Comment:
reading the spec: looks like the addtl_consent value is never an empty string. Could you please verify? If not, we may need to handle empty string specially.

Reply:
A valid `addtl_consent` string will never be empty. We do not need to handle this case explicitly.

Reply:
Sounds good. to confirm empty string will be converted to `undefined` internally and vendors won't be able to tell the difference

Reply:
Yup!